cookbooks/aws-parallelcluster-common/resources/network_service/partial/_network_service_debian.rb

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and limitations under the License.
 
 action :reload do
+  return if virtualized?
```

Comment:
Nitpick: please prefer `unless on_docker?` appended to the instruction you want to skip (in this case, to the  `end` of the execute command).